+find past EOF
+symbols index
+symbol find (<space>+f)
+fuzzy search
+build support
+git support
+location list (<space>+h/l)
+system clipboard (<space>+c/v)
+load texture
+ change shader to use font texture
--- 2022-10-25
+read fnt
+read tga
--- 2022-10-24
-export bitmap font: use bmfont
-change colors back
--- 2022-10-23
-cache culled characters
-try always rendering
--- 2022-10-22
-tilda support
-remove modified bool (use text compare instead)
-full insert undo
-dot support
-do not up cursor when zz at beginning of file
-fix dot and undo
--- 2022-10-21
-e-ink color scheme
-space whitespace mark
